util/caller/resolver.go

```diff
@@ -45,8 +45,20 @@ type CallResolver struct {
 }
 
 // defaultPattern strips src/github.com/organization/project/module/submodule/file.go
-// to module/submodule/file.go.
-var defaultPattern = "src/" + strings.Repeat(strings.Join([]string{"[^", "]+", ""}, string(os.PathSeparator)), 3) + "(.*)"
+// down to module/submodule/file.go.
+var defaultPattern = func() string {
+	_, file, _, ok := runtime.Caller(0)
+	if !ok {
+		panic("unable to look up location")
+	}
+	cur, err := filepath.EvalSymlinks(filepath.Dir(file))
+	const sep = string(os.PathSeparator)
+	if err != nil {
+		panic(err)
+	}
+	path := filepath.Join(cur, "../../../../..") + sep // $GOPATH/src/
```

Comment:
All those slashes need to be `sep` instead if we're going to care about `path` versus `filepath`.